src/js/hmm/components/__tests__/HMMList.test.tsx

```diff
@@ -0,0 +1,138 @@
+import { screen } from "@testing-library/react";
+import { connectRouter } from "connected-react-router";
+import { createBrowserHistory } from "history";
+import nock from "nock";
+import React from "react";
+import { combineReducers } from "redux";
+import { describe, expect, it } from "vitest";
+import { createFakeAccount, mockGetAccountAPI } from "../../../../tests/fake/account";
+import { createFakeHMMSearchResults, mockApiGetHmms } from "../../../../tests/fake/hmm";
+import { createGenericReducer, renderWithRouter } from "../../../../tests/setupTests";
+import { AdministratorRoles } from "../../../administration/types";
+import HMMList from "../HMMList";
+
+function createReducer(state, history) {
+    return combineReducers({
+        hmms: createGenericReducer(state.hmms),
+        tasks: createGenericReducer(state.tasks),
+        router: connectRouter(history),
+    });
+}
+
+describe("<HMMList />", () => {
+    const history = createBrowserHistory();
+    const fakeHMMData = createFakeHMMSearchResults();
+    let state;
+
+    beforeEach(() => {
+        state = {
+            hmms: fakeHMMData,
+            tasks: {
+                documents: [
+                    {
+                        complete: false,
+                        id: 21,
+                        progress: 33,
+                        step: "decompress",
+                    },
+                ],
+                id: 1,
+                progress: 10,
+            },
+        };
+    });
+
+    afterEach(() => nock.cleanAll());
+
+    it("should render correctly", async () => {
+        const scope = mockApiGetHmms(fakeHMMData);
+        renderWithRouter(<HMMList />, {}, history);
+
+        expect(await screen.findByText("HMMs")).toBeInTheDocument();
+        expect(screen.getByPlaceholderText("Definition")).toBeInTheDocument();
+
+        expect(screen.getByText(fakeHMMData.documents[0].cluster)).toBeInTheDocument();
+        expect(screen.getByText(fakeHMMData.documents[0].names[0])).toBeInTheDocument();
+
+        scope.isDone();
```

Comment:
```suggestion
        scope.done();
```
? I think `scope.isDone()` just returns a bool, whereas `scope.done()` should fail the test if not mocks haven't been called